pkg/apis/hive/v1/clusterdeployment_types.go

```diff
@@ -313,6 +317,9 @@ const (
 
 	// DeprovisionLaunchErrorCondition is set when a cluster deprovision fails to launch.
 	DeprovisionLaunchErrorCondition ClusterDeploymentConditionType = "DeprovisionLaunchError"
+
+	// ProvisionStoppedCondition is set when cluster provisioning is stopped
+	ProvisionStoppedCondition ClusterDeploymentConditionType = "ProvisionStopped"
```

Comment:
personally i do not think `ProvisionStopped` is a very user friendly condition name, because it does very little to explain why the provision was stopped.
- was it because some pre requisites were failed
- was it because we know retrying the install will not succeed, like when we know the creds or limit has caused the failure
- was it just because the retry budget was completed.

I would recommend we maybe start with some thing like `MaxRetryLimitReachedForProvisioning` maybe?

Reply:
based on https://github.com/openshift/hive/pull/1165/files#diff-5fc238b39e2a1fc17bd412fbac2d14eab4e0f205c91886b5671611bec4c665ccR645 and https://github.com/openshift/hive/pull/1165/files#diff-5fc238b39e2a1fc17bd412fbac2d14eab4e0f205c91886b5671611bec4c665ccR664

I can see this condition name being good enough.